_posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "2018-11-30-Reading: How the words Class-solidification becoming popular"
+date:   2018-11-30 19:31:07 +0800
+categories: readingnote
+---
+
+[Article Link](2018-11-30-Reading: The-dilemma-of-Class-solidification)
```

Comment:
Please state your sources more clearly. If it's a website, please include the URL to the article, if it's a book, perhaps include the author.

Reply:
It's a TED I watched, I have attached the video link.

Reply:
It'a a viewpoint from a book named Homo Deus, A Brief History of Tomorrow. I just add it to the article.